modules/UI/videolayout/VideoContainer.js

```diff
@@ -259,6 +269,18 @@ export class VideoContainer extends LargeContainer {
         this._resizeListeners = new Set();
 
         this.$video[0].onresize = this._onResize.bind(this);
+
+        if (isTestModeEnabled(APP.store.getState())) {
+            const cb = name => APP.store.dispatch(updateLastLargeVideoMediaEvent(name));
+            const containerHandlers = {};
+
+            containerEvents.forEach(event => {
+                containerHandlers[event] = cb.bind(this, event);
+            });
+            containerEvents.forEach(event => {
```

Comment:
I don't think you need a second loop here. You can add the code for adding a listener in the previous one, isn't it?

Reply:
🤦

Reply:
1:1 copy paste from lib-jitsi-meet 😉